lib/src/step.dart

```diff
@@ -292,6 +295,7 @@ class OnboardingStep {
     Color? pulseInnerColor,
     Color? pulseOuterColor,
     TapCallback? onTapCallback,
+    Function(VoidCallback nextStep)? onShowStep,
```

Comment:
I would prefer if you create a new typedef for this callback, named ShowStepCallback, I would would prefer consistent naming of its argument, e.g. in TapCallback it is called "next", I would like this here to be also "next" and not "nextStep"